account_invoice_line_sequence/__manifest__.py

```diff
@@ -16,6 +16,9 @@
     'data': ['views/account_invoice_view.xml',
              'views/report_invoice.xml'],
     'depends': ['account'],
+    'excludes': [
```

Comment:
Why the 2 modules cannot work together?

Reply:
I wrote it in the commit message:
> This module is not compatible with account_invoice_line_sequence because them both add to the context of invoice lines in invoice form view